src/mount_efs/__init__.py

```diff
@@ -1062,6 +1063,10 @@ def get_init_system(comm_file='/proc/1/comm'):
                 init_system = f.read().strip()
         except IOError:
             logging.warning('Unable to read %s', comm_file)
+
+        if init_system == 'init':
+            if os.path.isfile('/sbin/openrc'):
```

Comment:
Can combine these into one line

```python
if init_system == 'init' and os.path.isfile('/sbin/openrc'):
   ...
```

Also can you add some logging here that indicate we hack the init_system to be `openrc-init`?

Additionally, can you add some comment on top that introduce that this is to support openrc or something like that, so we know why we are doing this.

Reply:
thanks for (finally!) getting at this!  :)

i will double-check the patch vs. the latest released Alpine 3.15 cloud images to make sure this patch is still correct on that end, and fix the above.

Reply:
Requested changes should be completed now.